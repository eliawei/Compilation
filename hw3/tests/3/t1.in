// check in relop
void main() {
    int x = 5;
    byte y = 10b;
    printi(x + 4);
    set[5..260] myset1;
    bool z = y in myset1;
    z = x in myset1;
}
