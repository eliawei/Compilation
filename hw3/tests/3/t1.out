---end scope---
x INT 0
y BYTE 1
myset1 SET 2
z BOOL 3
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
main ()->VOID 0
